src/main/java/dao/FileAccessDaoCsvImpl.java

```diff
@@ -0,0 +1,47 @@
+package dao;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileAccessDaoCsvImpl implements FileAccessDaoCsv {
+
+    @Override
+    public List<String> readFromFile(String filePath) {
+        try {
+            return Files.readAllLines(Path.of(filePath));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file located at: " + filePath);
+        }
+    }
+
+    @Override
+    public void writeToFile(String stringForReport, String outFilePath) {
+        File toFile = new File(outFilePath);
+        ifExists(toFile);
```

Comment:
can we just use file.exists?

Reply:
thx, wasn't aware of the method until started "tests" HW. 
Removed this check for current task as non-relevant.